src/main/features/linux/mprisService.js

```diff
@@ -7,15 +7,20 @@ function mprisService() {
   let _songInfo = {};
 
   const player = mpris({
-    name: 'google-play-music-desktop-player',
+    name: 'google_play_music_desktop_player',
```

Comment:
I was under the impression that dashes were not allowed in bus names, but I just looked this up and they are [allowed but discouraged](https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names-bus). So I can change it back to what it was.